codespell_lib/data/dictionary.txt

```diff
@@ -7173,6 +7180,7 @@ Massachussets->Massachusetts
 Massachussetts->Massachusetts
 massmedia->mass media
 masster->master
+mast->must
```

Comment:
`mast` is a valid word but it could also be a typo. Perhaps adding:  
`mast->mast, must,`

Reply:
Removed